vendor/bat-native-ledger/include/bat/ledger/ledger.h

```diff
@@ -102,6 +102,14 @@ using GetBraveWalletCallback = std::function<void(type::BraveWalletPtr)>;
 
 using GetTransferableAmountCallback = std::function<void(double)>;
 
+using PostSuggestionsClaimCallback =
+    std::function<void(const type::Result result, std::string drain_id)>;
+
+using GetDrainCallback =
+    std::function<void(const type::Result result,
+                       std::string drain_id,
+                       std::string status)>;
```

Comment:
Perhaps we should make this an mojo-defined enum?

Reply:
Since the caller already knows the `drain_id`, can we simply callback with the `status`?

Reply:
agreed on the enum. done